testsuites/listener/shared/performance/test_performance_cbl_query.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import time
+
+from keywords.MobileRestClient import MobileRestClient
+from keywords.utils import log_info
+from keywords.document import create_docs
+# This test has to run when new ios version arrives atleast once.
+
+
+@pytest.mark.listener
+def test_view_query_performance(setup_client_syncgateway_test):
+    """
+     Run this test when new iOS version arrives to make sure CBL query is not diminishing
```

Comment:
s/to make sure CBL query is not diminishing/to make sure CBL query performance is not diminishing/

Reply:
done